README.md

```diff
@@ -1,2 +1,56 @@
-# community
-Longhorn Community
+# Longhorn Community
+
+Focuses on supporting the Longhorn Cloud native distributed block storage built on and for Kubernetes
+
+## Meetings
+* Regular Meeting: [Every 2nd Friday of the Month at 12:00 Eastern Standard Time (EST)](https://zoom.us/j/710325604) (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=12:00&tz=Eastern%20Standard%20Time%20%28EST%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1QD119jBX1h544ZQ9E4XaititEMfq8qEByx-42MtRC-g/).
+  * [Meeting recordings](https://www.youtube.com/channel/UCGk1f-LCVmccf1pX4OvF1Ig).
+
```

Comment:
Do we have an easy way to add a calendar invitation?

Reply:
The link `https://raw.githubusercontent.com/longhorn/community/master/Longhorn%20Community%20Meetup.ics` opens up a raw file on my computer. People may be confused by it. Not sure if we can get something like https://goo.gl/eyutah